rules/integrations/aws_bedrock/aws_bedrock_guardrails_multiple_violations_by_single_user.toml

```diff
@@ -80,11 +80,30 @@ timestamp_override = "event.ingested"
 type = "esql"
 
 query = '''
-from logs-aws_bedrock.invocation-*
+FROM logs-aws_bedrock.invocation-*
+
+// Filter for compliance violations detected
 | where gen_ai.compliance.violation_detected
-| keep user.id, gen_ai.request.model.id, cloud.account.id
-| stats violations = count(*) by user.id, gen_ai.request.model.id, cloud.account.id
-| where violations > 1
-| sort violations desc
+
+// Keep relevant ECS + model fields
+| keep
```

Comment:
Any reason we're mixing case for ES|QL functions? `FROM` vs `keep`

Reply:
Made all function calls and logical operators lowercase.